numba/cuda/tests/cudapy/test_powi.py

```diff
@@ -61,6 +71,32 @@ def test_powi(self):
         kernel[1, A.shape](A, power, Aout)
         self.assertTrue(np.allclose(Aout, A ** power))
 
+    def test_powi_int64(self):
+        dec = cuda.jit(void(float64[:, :], int64, float64[:, :]))
+        kernel = dec(cu_mat_power_v2)
+
+        A = np.arange(10, dtype=np.float64).reshape(2, 5)
```

Comment:
Why use `float64` data for what (appears to be / I would expect to be) to be a test of integer functionality?

Reply:
@gmarkall I used float64 because the typing for math.pow in cudaimpl.py the first argument is either a floating point value or will be converted to a floating point value. In particular the test case that is failing is a float64 first argument and a int64 second argument. I wanted to isolate the exact typing for this test case.

Reply:
Can you show me the test case that is failing? I don't see a use of `float64` in #8218, and this test case seems not to fail on `main`, so I'm not sure which test case you're referring to.